kafka-client/src/main/java/dev/responsive/kafka/internal/metrics/MetricPublishingCommitListener.java

```diff
@@ -45,9 +46,8 @@
  */
 public class MetricPublishingCommitListener implements Listener, Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(MetricPublishingCommitListener.class);
-  private final Metrics metrics;
+  private final ResponsiveMetrics metrics;
   private final String threadId;
-  private final String consumerGroup;
```

Comment:
FYI I didn't remove this tag just moved it to ResponsiveMetrics so now all metrics will be tagged with the group